automation/network-security-group/create-nsg-v2.ps1

```diff
@@ -26,17 +28,26 @@ param(
 
     [Parameter(Mandatory=$True)]
     [string]
-    $Location,            #Resource group location
+    $LocationForNetworkSecurityGroup,     #Resource group location
 
     [Parameter(Mandatory=$True)]
     [string]
-    $NetworkSecurityGroupName  #Network Secutiry Name to create
+    $TemplateJsonFilePath  #Path of the template.json file, either absolute or relative
```

Comment:
What happens if a user enters incorrect file path?

Reply:
oh very good question, i will look into it. thank you

Reply:
i added a validation for template.json file path input